src/kili/mutations/asset/__init__.py

```diff
@@ -322,13 +330,17 @@ def delete_many_from_dataset(
         asset_ids: Optional[List[str]] = None,
         external_ids: Optional[List[str]] = None,
         project_id: Optional[str] = None,
+        *,
+        wait_until_sucess: Optional[bool] = True,
     ) -> Asset:
         """Delete assets from a project.
 
         Args:
             asset_ids: The list of asset internal IDs to delete.
             external_ids: The list of asset external IDs to delete.
             project_id: The project ID. Only required if `external_ids` argument is provided.
+            wait_until_sucess: If True, the method will wait until the assets are fully deleted (synchronous mode).
+                If False, the method will return faster but the assets may not be fully deleted (asynchronous mode).
```

Comment:
I did the test on the SDK and it seems that even for asynchronous processes in the app, the SDK still wait until the end of the process to give the hand back